lib/core/common/widgets/custom_button.dart

```diff
@@ -52,30 +55,37 @@ class _CustomButtonState extends State<CustomButton> {
         width: widget.width,
         height: widget.height ?? 68,
         decoration: BoxDecoration(
-          color: Color(0xFF0D2231),
+          color: widget.isEnabled ? AppColor.accentColor : AppColor.grey,
           boxShadow: isPressed
               ? []
               : [
                   BoxShadow(
-                    color: Color(0xFF53D46B),
+                    color:
+                        widget.isEnabled ? AppColor.emeraldGreen : AppColor.mud,
                     offset: Offset(4, 4),
-                  )
+                  ),
+                  BoxShadow(
```

Comment:
the below 2 Box Shadow is having same offset as the above one, can remove 2nd and 3rd Box Shadows